creator-node/src/utils/legacyUtils.js

```diff
@@ -1,452 +0,0 @@
- * @returns {Boolean} returns true if the file was found in the network
- */
-export async function findCIDInNetwork(
-  filePath,
-  cid,
-  logger,
-  libs,
-  trackId = null,
-  excludeList = [],
-  numRetries = 5
-) {
-  if (!config.get('findCIDInNetworkEnabled')) return false
-
-  const attemptedStateFix = await getIfAttemptedStateFix(filePath)
-  if (attemptedStateFix) return false
-
-  // Get all registered Content Nodes
-  const creatorNodes = await getAllRegisteredCNodes(libs)
-  if (!creatorNodes.length) return false
-
-  // Remove excluded nodes from list of creator nodes or self, no-op if empty list or nothing passed in
-  const creatorNodesFiltered = creatorNodes.filter(
-    (c) =>
-      !excludeList.includes(c.endpoint) ||
-      config.get('creatorNodeEndpoint') !== c.endpoint
-  )
-
-  // Generate signature to auth fetching files
-  const delegateWallet = config.get('delegateOwnerWallet').toLowerCase()
-  const { signature, timestamp } = generateTimestampAndSignature(
-    { filePath, delegateWallet },
-    config.get('delegatePrivateKey')
-  )
-
-  let found = false
-  for (const { endpoint } of creatorNodesFiltered) {
-    if (found) break
-
-    try {
-      found = await asyncRetry({
-        asyncFn: async (bail) => {
-          let response
-          try {
-            response = await axios({
-              method: 'get',
-              url: `${endpoint}/file_lookup`,
-              params: {
-                filePath,
-                timestamp,
-                delegateWallet,
-                signature,
-                trackId
-              },
-              responseType: 'stream',
-              timeout: 1000
-            })
-          } catch (e) {
-            if (
-              e.response?.status === 403 || // delist
-              e.response?.status === 401 || // unauth
-              e.response?.status === 400 || // bad req
-              e.response?.status === 404 // not found
-            ) {
-              bail(
-                new Error(
-                  `Content is not available with statusCode=${e.response?.status}`
-                )
-              )
-              return
-            }
-
-            throw new Error(
-              `Failed to fetch content with statusCode=${e.response?.status}. Retrying..`
-            )
-          }
-
-          if (!response || !response.data) {
-            throw new Error('Received empty response from file lookup')
-          }
-
-          await writeStreamToFileSystem(
-            response.data,
-            filePath,
-            /* createDir */ true
-          )
-
-          const CIDMatchesExpected = await verifyCIDMatchesExpected({
-            cid,
-            path: filePath,
-            logger
-          })
-
-          if (!CIDMatchesExpected) {
-            try {
-              await fs.unlink(filePath)
-            } catch (e) {
-              logger.error(`Could not remove file at path=${path}`)
```

Comment:
`path` is an import so I think this is a bug. In the refactored version I corrected it to `filePath`

Reply:
great catch